src/WebJobs.Script/StorageProvider/IAzureTableStorageProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Azure.Data.Tables;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface IAzureTableStorageProvider
+    {
+        IConfiguration Configuration { get; }
+
+        bool TryCreateHostingTableServiceClient(out TableServiceClient client);
+
+        bool TryCreateTableServiceClientFromConnection(string connection, out TableServiceClient client);
```

Comment:
Should we use the same name for both methods? `TryCreateTableServiceClient`. The second one is just another overload which takes the connection string.